pkg/registry/storage/volumeattachment/storage/storage.go

```diff
@@ -93,3 +93,12 @@ func (r *StatusREST) Update(ctx context.Context, name string, objInfo rest.Updat
 	// subresources should never allow create on update.
 	return r.store.Update(ctx, name, objInfo, createValidation, updateValidation, false, options)
 }
+
+// Implement ShortNamesProvider
+var _ rest.ShortNamesProvider = &REST{}
+
+// ShortNames implements the ShortNamesProvider interface. Returns a list of short names for a resource.
+func (r *REST) ShortNames() []string {
+	return []string{"va"}
```

Comment:
volume attachments is not an object I would expect admins to regularly need to access (compared to pvc, pv, sc).  @jsafrane wdyt?

Reply:
Currently, `kubectl` already supports `get/delete/describe volumeattachments`, this pr just adds a shortName.

Reply:
we don't typically assign shortnames to all resources, just ones expected to be frequently accessed via kubectl. volumeattachments are automatically created/updated/deleted, and do not typically need to be accessed via kubectl